grammar.js

```diff
@@ -1296,8 +1298,21 @@ module.exports = grammar({
     identifier: $ => /[a-zA-Zα-ωΑ-Ωµ_][a-zA-Zα-ωΑ-Ωµ\d_]*/,
 
     _reserved_identifier: $ => alias(choice(
-      'default',
-      'union'
+      "abstract",
+      "async",
+      "become",
+      "box",
+      "default",
+      "do",
+      "final",
+      "macro",
+      "override",
+      "priv",
+      "try",
+      "typeof",
+      "unsized",
+      "virtual",
+      "yield",
```

Comment:
This `_reserved_identifier` rule is not intended to contain a list of all Rust's keywords. The only reason it exists is to handle cases where certain tokens can occur in positions where they can function *both* as keywords and as identifiers.

For example, I think that `union` needs to be here because you can do this:

```rust
fn main() {
  // `union` used as an variable name. ok.
  union = 6;

  // `union` used in a declaration. also ok.
  union MyUnion {
    a: i64,
    b: bool
  }
}
```

I don't know that any of these other tokens needs to be listed here.